File: js/compleated/58.js
let sol = (s) => { 
    let length = 0
    for (let i = 1; i <= s.length; i++) {
        if (s[s.length - i] == " " && length != 0) {
            return length
        } else if (s[s.length - i] != " "){
            length++
        }
    }

    return length
}

let s = "s"
console.log(sol(s))

File: rust/examples/58.rs

fn main() {
    let s = "   fly me   to   the moon  ".to_string();

    println!("{:?}", sol(s));
}


fn sol(s: String) -> i32 {
    let mut len = 0;

    for i in s.chars().rev() {
        if i == ' ' && len != 0 {
            return len
        } else if i != ' ' {
           len += 1 
        }
    }

    len
}
